components/tools/OmeroJava/test/integration/LightAdminRolesTest.java

```diff
@@ -592,65 +426,47 @@ public void testEdit(boolean isSudoing, boolean permWriteOwned,
 
     /**
      * Test that an ImporterAs can
-     * chgrp on behalf of another user solely with <tt>Sudo</tt> privilege
-     * only when this user is a member of both original and target groups
-     * Also test that light admin can, having the <tt>Chgrp</tt>
-     * privilege move another user's data into another group whether the
-     * owner of the data is member of target group or not.
+     * chgrp on behalf of another user in cases where the user (owner of the date)
```

Comment:
fixed in https://github.com/pwalczysko/openmicroscopy/commit/010053aa61b4fdf8a5e1b648b1a8b2fd1b4ba4ee#diff-d91232ef1e245a87e311c4a79a2a5f77L435